platform/macos/display_server_macos.mm

```diff
@@ -2100,6 +2101,37 @@
 	return ret;
 }
 
+Ref<Image> DisplayServerMacOS::clipboard_get_image() const {
+	Ref<Image> image;
+	NSPasteboard *pasteboard = [NSPasteboard generalPasteboard];
+	NSString *result = [pasteboard availableTypeFromArray:[NSArray arrayWithObjects:NSPasteboardTypeTIFF, NSPasteboardTypePNG, nil]];
```

Comment:
`NSPasteboardTypeTIFF` can this cause issues with the `representationUsingType:NSPNGFileType` below?

Reply:
`NSBitmapImageRep::representationUsingType` should encode it to the selected type regardless of source type.